### PR TITLE
add role collector

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: Backoffice/Collector/RoleCollector.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\Collector;
+
```
Comment (itkg-alavieille) [inline Backoffice/Collector/RoleCollector.php]: remove this empty line

File: BackofficeBundle/DependencyInjection/Compiler/RoleCompilerPass.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    /**
+     * You can modify the container here before it is dumped to PHP code.
+     *
+     * @param ContainerBuilder $container
+     *
+     * @api
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $roles = array(
```
Comment (itkg-alavieille) [inline BackofficeBundle/DependencyInjection/Compiler/RoleCompilerPass.php]: how an integrator can add a role and delete one of those roles ?

Comment (nicolasThal) [inline BackofficeBundle/DependencyInjection/Compiler/RoleCompilerPass.php]: I will write a little doc about it :)

**Comment by nicolasThal** [comment]
@itkg-alavieille pr updated

**Comment by nicolasThal** [comment]
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/60
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/51
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1122

File: BackofficeBundle/DependencyInjection/Compiler/RoleCompilerPass.php (hunk 2 above)
Comment (itkg-ngilain) [inline BackofficeBundle/DependencyInjection/Compiler/RoleCompilerPass.php]: I'm not sure it's a good idea to allow an iintegrator to remove basic roles. Add new one ok, but remove, I'm not sure

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**